ramls/schemas/userdataimport.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "User Schema",
-  "description": "A user",
+  "description": "A user; the same as used for /users API but with \"requestPreference\" property and \"require\" restriction added, and \"patronGroups\", \"departments\", \"addressTypeId\" and \"preferredContactTypeId\" are names and not UUIDs.",
```

Comment:
Should these differences between the schema here and the user schema be noted on the properties that are part of this schema as well?